roles/openshift_health_checker/openshift_checks/mixins.py

```diff
@@ -2,17 +2,14 @@
 from openshift_checks import get_var
 
 
-class NotContainerized(object):
+class NotContainerizedMixIn(object):
```

Comment:
I'm a fan of going the hierarchy route. I like mixins but not when traditional 1:1 hierarchy works. I'll put together a PR after this to switch it up assuming there is no objections.